### DISCUSSION TITLE
Allow customizing maximum filename length

### DISCUSSION THREAD

**Discussion by dotsam** [pymedusa/Medusa#8070] [Ideas & Feature requests]
When using certain filesystems, or overlay filesystems, maximum file length can't be the standard 255 characters that is usually supported.

Code in in _format_pattern() currently  is hard-coded to 244 characters based on the assumption of 255 characters, and giving ~10 characters headroom for a potential `-thumb.jpg` suffix
https://github.com/pymedusa/Medusa/blob/d9c31da063e6c4b4383749e19b27b13693a1b71e/medusa/tv/episode.py#L1776

I'd like to see the ability to customize this length check, if not in the GUI, then at least as an advanced ini file only setting.

**Comment by nydave69** [comment]
I'd like to see a rename setting that would limit the number of characters in the episode name like %EN(60)..
It would allow the first 60 characters of the episode name added to the file name.
